BookingsAPI/Bookings.AcceptanceTests/Steps/HearingsSteps.cs

```diff
@@ -218,6 +218,10 @@ public void ThenHearingShouldBe(Domain.Enumerations.BookingStatus status)
             var model = RequestHelper.DeserialiseSnakeCaseJsonToResponse<HearingDetailsResponse>(_context.Response.Content);
             model.UpdatedBy.Should().NotBeNullOrEmpty();
             model.Status.Should().Be(status);
+            if(status == Domain.Enumerations.BookingStatus.Created)
+            {
+                model.ConfirmedBy.Should().NotBeNullOrEmpty();
```

Comment:
Hi Sunil, We can't do this check as the older hearings will have a null for this field.

Reply:
But this done against the hearing in question. This is run for all the hearings anyway.